src/dask_awkward/io.py

```diff
@@ -55,3 +52,36 @@ def from_parquet(source: Any, **kwargs: Any) -> DaskAwkwardArray:
 
     hlg = HighLevelGraph.from_collections(name, dsk)
     return new_array_object(hlg, name, None, npartitions=npartitions)
+
+
+def _chunked_to_json_v2(chunks):
+    return from_iter_v2(json.loads(ch) for ch in chunks.split(b"\n") if ch)
+
+
+def _chunked_to_json_v1(chunks):
+    return from_iter_v1(json.loads(ch) for ch in chunks.split(b"\n") if ch)
+
+
+def _from_json(
```

Comment:
This is implicitly (line-)delimited JSON, i.e., not a huge single JSON object. Not sure if we need to bother considering that latter possibility.

Reply:
now handling multiple cases (line delim (single and multi file), single file, multi file)